courses/machine_learning/cloudmle/cloudmle.ipynb

```diff
@@ -47,7 +47,7 @@
     "os.environ['PROJECT'] = PROJECT\n",
```

Comment:
<p>Please change the copyright to 2020.  Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Updated the same.